extensions/vscode/README.md

```diff
@@ -1,11 +1,68 @@
-# Posit Publisher
+# Posit Publisher - Deploy to Posit Connect
 
-[Posit Publisher](https://github.com/posit-dev/publisher) is an extension for
-[Visual Studio Code](https://code.visualstudio.com/) that provides an easy
-way to deploy to [Posit Connect](https://posit.co/products/enterprise/connect/).
+[Posit Publisher](https://github.com/posit-dev/publisher) is a code-first tool
+with a push-button user interface for deploying the things you build in R and
+Python to [Posit Connect](https://posit.co/products/enterprise/connect/).
+
+Supported content includes:
+
+- Python content: APIs, applications, and notebooks
+- Quarto content: `.qmd` and `.Rmd` files. Support for embedded Shiny apps
+- R content: Shiny apps, RMarkdown, Plumber APIs, and Rmd with embedded
+  Shiny apps
 
 ## Installation
 
-Install Posit Publisher from the
+Install Posit Publisher by using the **Install** button above, or using the
+**Extensions** side bar in [Positron](https://github.com/posit-dev/positron) or
+[VS Code](https://code.visualstudio.com/) and searching for `posit.publisher`.
+
+Posit Publisher is available from the
 [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=Posit.publisher)
 or from the [Open VSX Registry](https://open-vsx.org/extension/posit/publisher).
+
+## Features
+
+### Code-first deployment
+
+Configure your content and then deploy to Posit Connect with a click.
+
+[Configuration files](https://github.com/posit-dev/publisher/blob/main/docs/configuration.md)
+allow setting up how you want your content to look and run on Connect - from
+choosing a title and description to
+[Runtime process settings](https://docs.posit.co/connect/user/content-settings/#content-runtime).
+
+Keep your content stable with language and tool versions set in your
+configurations — all automatically detected based on your local environment.
+
+Everything is saved in the `.posit/` directory for easy re-deployment and
+collaboration.
+
+### Seamless collaboration
+
+Allow Collaborators to easily pick up where you left off by committing the
+`.posit/` directory to your version control system. Your team can deploy using
+the same deployments and configurations.
```

Comment:
IMHO, we might hold this for when we have a more dedicated story around this. I think the code / features are almost all there, but I'm not sure folks will read this and immediately get the process we are imagining here. It would be nice to have a how to or walkthrough describing that that we could link to.

Reply:
If I'm understanding the suggestion - this sounds like removing the "Seamless collaboration" feature section in this PR, and adding it in the future when we have those code / features and a document to walk users through the collaboration flow?

Reply:
Yeah, that's what I was thinking. It's not a "you should absolutely do that" but I was thinking about it and what I would as a user take away from this and I'm not sure there's enough detail here (or elsewhere) that really shows off what this allows (and writing that how to will take a second, probably not before 4.2)

Reply:
I removed the section, and added a follow-up task to document our collaboration flow. As a part of that we can add it back to the feature list in this README